tests/unit/components/materialize-button-dropdown-test.js

```diff
@@ -84,3 +84,33 @@ test('dropdown shown below origin', function(assert) {
     });
   });
 });
+
+test('dropdown with constrained width', function(assert) {
+  const component = this.subject();
+  component.set('constrainWidth', true);
+  this.render();
+
+  const dropdownContentId = `#${component.get('_dropdownContentId')}`;
+  const dropdownElement = $(dropdownContentId);
+  component.$().click();
+  Ember.run(function() {
+    Ember.run.schedule('afterRender', function() {
+      assert.equal(`${component.$().outerWidth()}px`, dropdownElement.css('width'));
```

Comment:
It doesn't actually, was just copied from tests above that also had the run loop. These were needed on an older version of the tests that were using `dropdownElement.position().top` instead of `dropdownElement.css('top')`, which wasn't updated until `afterRender`.